tests/utils.go

```diff
@@ -1411,25 +1411,11 @@ func DeleteAlpineWithNonQEMUPermissions() {
 }
 
 func renderContainerSpec(imgPath string, name string, cmd []string, args []string) k8sv1.Container {
-	nonRootUser := int64(1042)
```

Comment:
We only need to remove the user.

Reply:
ACK. [DONE](https://github.com/kubevirt/kubevirt/compare/9cd13d0ff17a90f6ada1bf994262ed9143b4c370..90e3f02200d9714eb5c5aa733cc4a09356c1a02e)